internal/feature_test.go

```diff
@@ -110,8 +111,17 @@ func TestScriptBasedInit(t *testing.T) {
 
 }
 
-// TestScriptBasedCore tests ScriptBasedSoftwareUpdatable core functionality: init, install and download.
-func TestScriptBasedCore(t *testing.T) {
+// TestScriptBasedDownloadAndInstall tests ScriptBasedSoftwareUpdatable core functionality: init, install and download operations.
+func TestScriptBasedDownloadAndInstall(t *testing.T) {
+	testScriptBasedSoftwareUpdatableOperations(true, t)
+}
+
+// TestScriptBasedDownloadAndInstallResume tests ScriptBasedSoftwareUpdatable core functionality: init, resuming install and download operations.
```

Comment:
`tests ScriptBasedSoftwareUpdatable core functionality` - should be: `tests ScriptBasedSoftwareUpdatable core functionality with a operation resume`